examples/vpc-saperate-private-db-cache-route/main.tf

```diff
@@ -0,0 +1,29 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+module "vpc" {
+  source = "./terraform-aws-vpc"
+
+  name = "sandbox"
+
+  cidr = "10.10.0.0/16"
+
+  azs                 = ["ap-southeast-1a", "ap-southeast-1b"]
```

Comment:
Pick AZs from `eu-west-1`, please. All other examples are using that region. Have at least 3 AZs in examples.

Reply:
Done, have used `eu-west-1` now and taken 3 AZs